prow/plugins/label/label.go

```diff
@@ -31,7 +31,7 @@ import (
 const pluginName = "label"
 
 var (
-	labelRegex              = regexp.MustCompile(`(?m)^/(area|committee|kind|language|priority|sig|triage|wg)\s*(.*)$`)
+	labelRegex              = regexp.MustCompile(`(?m)^/(area|committee|cuj|kind|language|platform|priority|sig|triage|wg)\s*(.*)$`)
```

Comment:
While we are here, can we also add `team`? I would like to use it on an external cluster.

Reply:
Done.